platform/security/sts/security-sts-propertyclaimshandler/src/test/java/org/codice/ddf/security/sts/claims/property/TestAttributeFileClaimsHandler.java

```diff
@@ -158,6 +160,42 @@ public void testRetrieveClaimsValuesNullPrincipal() {
     ProcessedClaimCollection processedClaims =
         attributeFileClaimsHandler.retrieveClaimValues(claimCollection, claimsParameters);
 
-    Assert.assertThat(processedClaims.size(), CoreMatchers.is(equalTo(0)));
+    assertThat(processedClaims, is(empty()));
+  }
+
+  @Test
+  public void testGetSingleValuedSystemHighAttribute() throws IOException {
+    assertThat(attributeFileClaimsHandler.getValues("test"), containsInAnyOrder("testValue"));
+  }
+
+  @Test
+  public void testGetMultiValuedSystemHighAttribute() throws IOException {
+    assertThat(
+        attributeFileClaimsHandler.getValues("test1"),
+        containsInAnyOrder("testing1", "testing2", "testing3"));
```

Comment:
❗️ Since we're returning a `List`, we should validate the order as well.

Reply:
See my other reply about updating the `getValues` method to return a `Set<String>` instead.